packages/truffle/test/scenarios/commands/develop.js

```diff
@@ -25,20 +25,29 @@ describe("truffle develop", function () {
   });
 
   describe("Globals", function () {
+    const cunningWord = "contrafibularities";
     let output;
+
     before(async function () {
       this.timeout(10000);
-      const input = "Object.keys(global)";
+      const inputs = [`dict = "${cunningWord}"`, "Object.keys(global)"];
 
       await CommandRunner.runInREPL({
-        inputCommands: [input],
+        inputCommands: inputs,
         config,
         executableCommand: "develop",
         displayHost: "develop"
       });
       output = logger.contents();
     });
 
+    it("Sends multiple commands to REPL", () => {
```

Comment:
I landed on placing a test here because it's quicker to piggyback in this scenario than breakout another file to validate our testing library. Please let me know if you think this belongs elsewhere.